backends/p4tools/modules/smith/common/expressions.cpp

```diff
@@ -707,7 +712,7 @@ IR::Expression *ExpressionGenerator::constructCmpExpr() {
     // gen some random type
     // can be either bits, int, bool, or structlike
     // for now it is just bits
-    auto newTypeSize = Utils::getRandInt(1, 128);
+    auto newTypeSize = Utils::getRandInt(1, 64);
```

Comment:
Please add a TODO here to make the bitwidth a generic parameter. 
Maybe MAX_BITWIDTH.